### PR TITLE
Release Linux/SQL project

### PR DESCRIPTION
**Pull request by Kelldav**
Setting Linux/SQL project ready for release and review.

### CODE REVIEW

File: linux_sql_core_java/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+To test my bash scripts and sql queries I simply gathered data within my tables and used that as testing data. For the averages I also used traditional methods of manually adding and dividing the values in order to make sure that the result produced is accurate.
+
+# Deployment
+To deploy the app I used GitHub to store the files, crontab to automate the execution of the script and docker to contain the instance of psql. This way the program will not be too demanding and operate with ease.
+#Improvements
+-Improved user experience.
+
+-Add additional queries to help user find key information.
+
+-Improve README with a Frequently Asked Questions section.
```
Comment (jarviscanada) [inline linux_sql_core_java/README.md]: missing a space after `-` for those bullet points

File: linux_sql_core_java/scripts/host_info.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#output to psql
+psql -h $psql_host -p $psql_port -d $db_name -U $psql_user -w << EOF
+\dt
+\c host_agent
+\dt
+INSERT INTO host_info VALUES ('1','$hostname','$cpu_number','$cpu_architecture','$cpu_model','$cpu_mhz','$l2_cache','$total_mem','$timestamp');
+SELECT *FROM host_info
+EOF
+exit 0
```
Comment (jarviscanada) [inline linux_sql_core_java/scripts/host_info.sh]: exit $?

File: linux_sql_core_java/scripts/host_usage.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+echo disk_available
+#output to psql
+psql -h $psql_host -p $psql_port -d $db_name -U $psql_user -w << EOF
+\dt
+\c host_agent
+\dt
+INSERT INTO host_usage VALUES ('$timestamp','1','$memory_free','$cpu_idle','$cpu_kernel','$disk_io','$disk_available');
+SELECT *FROM host_usage
+EOF
+
```
Comment (jarviscanada) [inline linux_sql_core_java/scripts/host_usage.sh]: exit $?

**Review by jarviscanada — APPROVED** [review]
LGTM

**Comment by Kelldav** [comment]
Closing due to more Work in progress code within Develop Branch. Another Pull request will be made later.

**Pull request closed without merging**